tests/Doctrine/Tests/Common/Annotations/DocParserTest.php

```diff
@@ -1258,16 +1258,6 @@ public function testSetValuesExeption()
         $this->createTestParser()->parse($docblock, 'some class');
     }
 
-    /**
-     * @expectedException \Doctrine\Common\Annotations\AnnotationException
-     * @expectedExceptionMessage [Syntax Error] Expected Doctrine\Common\Annotations\DocLexer::T_IDENTIFIER or Doctrine\Common\Annotations\DocLexer::T_TRUE or Doctrine\Common\Annotations\DocLexer::T_FALSE or Doctrine\Common\Annotations\DocLexer::T_NULL, got '3.42' at position 5.
-     */
-    public function testInvalidIdentifierInAnnotation()
```

Comment:
Urgh, I shouldn't have removed this test. It should at least have a test declaring another sort of failure here.